src/reference/informative/informA2_6.rst

```diff
@@ -0,0 +1,97 @@
+.. _inform2_6:
+
+.. container:: toggle
+
+  .. container:: header
+
+    See more
+
+  .. container:: infospec
+
+    .. container:: heading3
+
+      Understanding element IDs
+
+    In order to understand how the XML :code:`id` attribute is used in CellML models, we need to remember three things.
+    Firstly, in CellML the elements are identified by uniqueness over the *model* scope of their :code:`name` attribute, their type (eg: :code:`variable`), and in some cases, by their parent element too.
+    Secondly, the CellML :code:`model` might exist over several different *documents*, and make use of :code:`import` functionality to connect these documents into one *model*.
+    Finally, the uniqueness of the XML :code:`id` attribute value extends only to the *document* scope, not the CellML *model* scope.
+    Consider the example below.
+
+    .. code::
+
+      # In the document "BreakfastMenu.cellml".
+      # All id attributes are unique within this document.
+      model: name = "BreakfastOfChampions", id = "b"
+        ├─ component: name = "Protien", id = "p"
+        ├─ component: name = "Carbohydrates", id = "c" <╴╴╴╴╴╴╴╴╴┐
+        └─ component: name = "FunStuff", id = "f"                ╷
+             └─ variable: name = "FunStuff", id = "fs"           ╷
+                                                                 ╷
+                                              Valid: Component named "Porridge" is
+                                              imported, but its id attribute "p" is not.
```

Comment:
I'm not 100% sure about this sentence.

To me the issue is more that the import happens at a conceptual level, so no XML is ever "imported".
And the `id` doesn't exist at the conceptual level: it's part of the XML document, but not part of the CellML model it represents. Similar to the `<` and `>` characters, or whitespace, or XML comments.

The import happens _after parsing_, when ids, comments, etc. no longer exist

Reply:
I've had a go in 0fe0334 ... not sure if it's right yet though.